ui/models/user.js

```diff
@@ -8,7 +8,8 @@ export default class UserModel {
   }
 
   static get() {
-    return fetch(`${uri}`)
+    // TODO: This credentials value can be removed in July once Firefox 62 ships and it is the default.
+    return fetch(`${uri}`, { credentials: "same-origin" })
```

Comment:
I think the methods in `ui/helpers/http.js` will be affected too.

Reply:
Crap.  You're right, of course.  Thanks!